src/Sylius/Bundle/ApiBundle/Validator/Constraints/ShipmentNotShippedStateValidator.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Validator\Constraints;
+
+use Sylius\Bundle\ApiBundle\Command\Checkout\ShipShipment;
+use Sylius\Component\Core\Model\ShipmentInterface;
+use Sylius\Component\Core\OrderShippingStates;
+use Sylius\Component\Core\Repository\ShipmentRepositoryInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Webmozart\Assert\Assert;
+
+final class ShipmentNotShippedStateValidator extends ConstraintValidator
```

Comment:
```suggestion
final class ShipmentAlreadtShippedValidator extends ConstraintValidator
```

Reply:
`ShipmentAlreadtShippedValidator` -> `ShipmentAlreadyShippedValidator` 💃 😄

Reply:
Never drink and code :D